README.md update with description of issues.md (#2853)
SonarSource/sonar-dotnet: docs/issues.md

# Issue tracking

All issues for SonarC\# and SonarVB are tracked using [GitHub Issues](https://github.com/SonarSource/sonar-dotnet/issues).

> You're welcome to submit any issue that you've found using these plugins. Please provide minimal reproducible example for each case.

Reviewer: ```suggestion
You're welcome to submit any issue that you've found using the C# and VB.NET code analyzers (as [standalone nugets](https://www.nuget.org/packages/SonarAnalyzer.CSharp/), inside [SonarLint for Visual Studio](https://www.sonarlint.org/visualstudio/), as [SonarQube](https://www.sonarqube.org/) plugins or in [SonarCloud](https://sonarcloud.io/about) analysis).

Please provide a minimal reproducible example in each case.
```

Reviewer: (note: in the above change suggestions there are two sides
- adding the promotion :) and mentioning the code analyzers rather than the plugins
- small grammar fix "Please provide **a** minimal reproducible example"

## Labels

> Labels are divided into several categories. Single issue can contain:

Reviewer: ```suggestion
Labels are divided into several categories. One issue can be labelled with:
```
